src/osd/OSD.cc

```diff
@@ -2228,6 +2228,46 @@ int OSD::mkfs(CephContext *cct,
   return ret;
 }
 
+int OSD::run_osd_bench(CephContext *cct,
```

Comment:
Do you really need the oss as an out parameter?
Maybe return a tl::expected<string, int> instead?

Reply:
Yes, we can do without oss. I have used tl::expected as per your suggestion. Thanks!